xml_inscriptions/transcribed/MA.Glouc.HCM.L.03.8.3.xml

```diff
@@ -75,23 +75,24 @@
                                         <width>24</width>
                                     </dimensions>
                                 </support>
-                                <condition ana="#complete.intact">
-                                    <p>Bottom corners show evidence of having been cut to fit into a slot</p>
+                                <condition ana="#complete"> <!-- do i need to keep the .intact? -->
```

Comment:
I'd keep the intact unless you're correcting the work; I think I originally assigned the granularity of the control vocabulary based on my inspection of the images and files.

Reply:
I think that the issue was that the "complete" applies to the inscription itself, while the "intact: applies to the support piece, so I ended up deciding to only keep "#complete" and specify that the support was intact in the description box, which come up again later.